src/pykeen/evaluation/evaluator.py

```diff
@@ -531,7 +532,9 @@ def evaluate(
 
     # Prepare batches
     if batch_size is None:
-        batch_size = 1
+        # This should be a reasonable default size that works on most setups while being faster than batch_size=1
+        batch_size = 32
```

Comment:
same here - if they user set AMO to true, they might expect it gets run and could be confused by the logging that a batch_size was not provided

Reply:
I added some info statements.